scipy/signal/fir_filter_design.py

```diff
@@ -583,3 +583,84 @@ def remez(numtaps, bands, desired, weight=None, Hz=1, type='bandpass',
     bands = np.asarray(bands).copy()
     return sigtools._remez(numtaps, bands, desired, weight, tnum, Hz,
                            maxiter, grid_density)
+
+
+def firls(numtaps, bands, desired, weight=None, Hz=1):
```

Comment:
Using `Hz` as the argument name is consistent with `remez`, but it's not a great name for this argument.  This is a case where I think we should use a better name instead of using a bad name for the sake of consistency.  `coherence`, `csd`, `periodogram`, and `welch` use `fs` for the sampling frequency, while `firwin` and `firwin2` use `nyq` for the Nyquist frequency.  My preference is `fs`, but either is better than `Hz`.